vcsclient/vcsclient.go

```diff
@@ -209,6 +209,8 @@ type VcsClient interface {
 
 	// GetRepositoryEnvironmentInfo Gets the environment info configured for a repository
 	GetRepositoryEnvironmentInfo(ctx context.Context, owner, repository, name string) (RepositoryEnvironmentInfo, error)
+	// GetModifiedFiles returns list of file names modified between two commit SHA.
+	GetModifiedFiles(ctx context.Context, owner, repository, beforeSha, afterSha string) ([]string, error)
```

Comment:
Missing parameters description (in GetRepositoryEnvironmentInfo too)

Reply:
The `GetRepositoryEnvironmentInfo` is not part of this change, so nothing to do with it here.